src/Microsoft.Diagnostics.EventFlow.ServiceFabric/ConfigurationProvider/SecureStringExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+
+namespace Microsoft.Extensions.Configuration.ServiceFabric
+{
+    public static class SecureStringExtensions
+    {
+        public static string ToUnsecureString(this SecureString secureString)
+        {
+            if (secureString == null)
```

Comment:
Fixed. This code was copied from somewhere else and I did not pay enough attention, good catch!